algorithms/class2_homework.html

```diff
@@ -0,0 +1,185 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+
+    <title>Homework 2</title>
+
+    <meta name="description" content="">
+    <meta name="author" content="Pamela Fox">
+
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+
+    <link rel="stylesheet" href="../common/css/reveal.min.css">
+
+    <!-- FOR LIGHT -->
+    <link rel="stylesheet" href="../common/css/gdilight.css" id="theme">
+    <style>
+        .highlight { color: #2a4758 !important; }
+    </style>
+
+    <link rel="stylesheet" href="css/slides.css">
+    <!-- If the query includes 'print-pdf', include the PDF print sheet -->
+    <link rel="stylesheet" href="css/algorithms.css">
+
+    <script>
+    if (window.location.search.match(/print-pdf/gi)) {
+        var link = document.createElement('link');
+        link.rel = 'stylesheet';
+        link.type = 'text/css';
+        link.href = 'reveal/css/print/pdf.css';
+        document.getElementsByTagName('head')[0].appendChild(link);
+    }
+    </script>
+    <link rel="icon" type="image/x-icon" href="../common/favicon.ico" />
+    <!--[if lt IE 9]>
+    <script src="lib/js/html5shiv.js"></script>
+    <![endif]-->
+
+
+</head>
+
+<style>
+    .left { text-align: left; }
+</style>
+
+<body>
+
+    <div class="reveal">
+
+        <!-- Any section element inside of this container is displayed as a slide -->
+        <div class="slides">
+            <!-- Opening -->
+            <section>
+                <h1>Class 2 Homework</h1>
+            </section>
+
+            <section>
+                <h2>Matrices</h2>
+                <p>Check given matrix is magic square or not</p>
+                <pre>
+                    <code>
+Input : n = 3
+  2   7   6
+  9   5   1
+  4   3   8
+Output : Magic matrix
+Explanation:In matrix sum of each
+row and each column and diagonals sum is 
+same = 15.
+
+Input : n = 3
+  1   2   3 
+  4   5   6
+  7   8   9
+Output : Not a Magic Matrix
+Explanation:In matrix sum of each
+row and each column and diagonals sum is
+not same.
+                    </code>
+                </pre>
+                <!--
```

Comment:
Are you not providing the link to the solution? If so, how do you plan to go through this with student?